ajax/libs/AvatarJS/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "AvatarJS",
+  "filename": "avatarServices.js",
+  "version": "0.2.0",
+  "description": "AvatarJS project for cdn",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/IPENGR/AvatarJS.git",
+    "basePath": "dist/",
+    "files": [
+      "avatarServices.js"
+    ]
+  },
+  "author": "Innovapath",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/IPENGR/AvatarJS.git"
+  },
+
+  "dependencies": {
+
+  },
+  "devDependencies": {
```

Comment:
We don't need `devDependencies` in package.json. I think we can remove this field.